mk/source.mk

```diff
@@ -233,7 +233,7 @@ COMMON_SRC = \
             drivers/light_ws2811strip.c \
             drivers/rangefinder/rangefinder_hcsr04.c \
             drivers/rangefinder/rangefinder_lidartf.c \
-            drivers/rangefinder/rangefinder_virtual.c \
+            drivers/rangefinder/rangefinder_lidarmt.c \
```

Comment:
Think rangefinder_virtual is used by SITL ?

Reply:
Sorry, what is SITL?
rangefinder_virtual is not used anywhere at the moment; it comes from INAV implementation, and I replaced it with a device-specific driver.

Reply:
Software in the loop: https://betaflight.com/docs/development/SITL

Reply:
Agree cannot find any reference in SITL, not even here https://github.com/betaflight/betaflight/pull/4753

Reply:
I don't think so, I added this file myself, and as you see, the pull request is not approved yet, 
If needed, we can implement it later for the simulation purpose.